src/ServiceProvider.php

```diff
@@ -40,7 +40,7 @@ private function getInstalledProviders()
 
         $providers = [];
         foreach ($config->require as $package => $version) {
-            if (preg_match('/^league\/oauth2-(?!client)(.+)$/', $package, $match)) {
+            if (preg_match('/oauth2-(?!client)(.+)$/', $package, $match)) {
                 $providers[] = $match[1];
```

Comment:
Ohhh clever that's cool. Still in favour or `use`. Feels less fragile, but does require another point of edit for a new provider.